central/declarativeconfig/utils/verify_origin.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+import (
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/declarativeconfig"
+	"github.com/stackrox/rox/pkg/errox"
+)
+
+func VerifyReferencedResourceOrigin(referencing, referenced declarativeconfig.ResourceWithTraits, referencingName, referencedName string) error {
```

Comment:
Just from the way my brain thinks: can we switch `referencing` with `referenced` in the order of params? I read this within the tests as `default -> imperative` etc. instead of the other way around.